src/yachtzee/index.ts

```diff
@@ -115,7 +114,7 @@ function _yachtzeeChain(): void {
         useSkill($skill`The Polka of Plenty`);
       }
     }
-    adventureMacro($location`The Sunken Party Yacht`, Macro.abort());
+    garboAdventure($location`The Sunken Party Yacht`, Macro.abort());
```

Comment:
is it worth making a special function for when we want to pass `Macro.abort()`?

Reply:
Just feels like it happens a decent amount.

Reply:
Yeah, but outside scope of this I think